addons/payment_sips/models/payment_transaction.py

```diff
@@ -0,0 +1,179 @@
+# coding: utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright 2015 Eezee-It
+import datetime
+from dateutil import parser
+import json
+import logging
+import pytz
+import re
+import time
+from hashlib import sha256
+
+from werkzeug import urls
+
+from odoo import models, fields, api
+from odoo.tools.float_utils import float_compare
+from odoo.tools.translate import _
+from odoo.addons.payment.models.payment_acquirer import ValidationError
+from odoo.addons.payment.utils import singularize_reference_prefix
+from odoo.addons.payment_sips.controllers.main import SipsController
+
+from .const import SIPS_SUPPORTED_CURRENCIES, SIPS_STATUS_CANCEL, SIPS_STATUS_PENDING, SIPS_STATUS_ERROR, SIPS_STATUS_REFUSED, SIPS_STATUS_VALID, SIPS_STATUS_WAIT
+
+_logger = logging.getLogger(__name__)
+
+
+class TxSips(models.Model):
```

Comment:
Nitpicking (same as above)
```suggestion
class PaymentTransaction(models.Model):
```